data/morphology/grab_unimorph_data.py

```diff
@@ -6,47 +6,82 @@
 import os
 import requests
 import time
-
+start_time = time.time()
 from typing import Dict
 
 
+
+
 UNIMORPH_DICT_PATH = "unimorph_languages.json"
 
 
-def download(data_to_grab: Dict[str, str]) -> Dict[str, str]:
-    to_retry = {}
-    os.mkdir("tsv")
-    for language, url in data_to_grab.items():
-        with requests.get(url, stream=True) as response:
-            logging.info("Downloading: %s", language)
+def download():
```

Comment:
It seems like most of this is a regression from previous version, I'll let you clean that up.